doc/STYLE_GUIDE.md

```diff
@@ -68,3 +68,5 @@
 
 [plugin]: http://editorconfig.org/#download
 [Oxford comma]: https://en.wikipedia.org/wiki/Serial_comma
+[emdashes]: https://en.wikipedia.org/wiki/Dash
```

Comment:
Nit: Perhaps this should link to https://en.wikipedia.org/wiki/Dash#Em_dash instead?